scripts/stackblitz.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+# Simplified start script for the stack blitz environment, which can't handle multiple watchers
+set -x
+set -e
+
+(cd packages/gestalt-design-tokens && yarn build)
+(cd packages/gestalt && yarn build)
+(cd packages/gestalt-charts && yarn build)
+(cd packages/gestalt-datepicker && yarn build)
+(cd docs && yarn build-metadata)
+(cd docs && (npx next dev))
```

Comment:
Only docs are live-reloaded when running in the stackblitz env. Other commands can be run as needed